contrib/sailfish/navit-sailfish.spec

```diff
@@ -130,9 +130,12 @@ fi
 
 
 %changelog
+<<<<<<< HEAD
```

Comment:
Please fix merge conflict. Just in the log, but...

Reply:
Resolved the merge conflict